instrumentation/database/sql/splunksql/config.go

```diff
@@ -0,0 +1,268 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package splunksql
```

Comment:
Can we rename the package to `otelsql` as it doesn't contain any Splunk- specific features?

Reply:
Can we move this conversation to https://github.com/signalfx/splunk-otel-go/pull/88 where this is defined?

I'm open to the discussion, but don't want to change that value here.

Reply:
Yeah. It was a mistake done from a mobile 😄 Resolving